src/lib/ledger_catchup/ledger_catchup.ml

```diff
@@ -4,6 +4,27 @@ open Protocols.Coda_transition_frontier
 open Pipe_lib
 open Coda_base
 
+(** [Ledger_catchup] is a process that connects a foreign external transition
+    into a transition frontier by asking it's peer a path of external_transitions.
+    It receives the external_transition from [Catchup_monitor]. With that
```

Comment:
for some clarity: `It receives the external_transition to catchup to from [Catchup_monitor]`.